LiteCore/Storage/RecordEnumerator.cc

```diff
@@ -82,7 +74,7 @@ namespace litecore {
                 close();
                 return false;
             }
-            LogToAt(EnumLog, Debug, "enum:     --> [%s]", _record.key().hexCString());
+            //LogToAt(QueryLog, Debug, "RecordEnumerator %p  --> '%.*s'", this, SPLAT(_record.key()));
```

Comment:
Is there a reason for leaving this in as a comment?

Reply:
It's very useful in specific circumstances, when debugging RecordEnumerator as I was yesterday, but it spews an AWFUL LOT of stuff during unit tests. I guess Debug logging isn't going to be on often, though, so I could leave it there.